src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <logo class="logo">
+        <a href="#index.html" class="logo__link">
+          <img
+           class="logo-img"
+           src="./images/logo.png"
+           alt="logo"
+           >
+        </a>
+      </logo>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops-computers"
+              class="nav__link"
+              data-qa = "hover"
+              >Laptops & Computers</a>
```

Comment:
```suggestion
            <a 
              href="#laptops-computers"
              class="nav__link"
              data-qa = "hover"
            >
              Laptops & Computers
            </a>
```
if you have a lot of attributes you should write all of them from the new line, also content of the tag is also better to write from the new line for better readability
13'th point in checklist